app/assets/javascripts/togglereviewformvisibility.js

```diff
@@ -1,11 +1,37 @@
-function toggleVisibility(id) {
-  var e = document.getElementById(id);
-  if(e.style.display == 'block')
-  {
-     e.style.display = 'none';
-   }
-  else
-  {
-     e.style.display = 'block';
-   }
-}
+$(function(){
+  $(".button-edit-form").on("click", function(){
+    var $panel = $(this).closest(".panel");
+    var $form = $panel.find(".form-div");
+    $form.toggleClass("hide");
+  });
+});
+
+$(function(){
+  $(".button-new-form").on("click", function(){
+    debugger;
+    var $div = $(this).closest(".new-review");
+    var $form = $div.find(".reviewform");
+    $form.toggleClass("hide");
+  });
+});
+
+$('.edit_button').click(function(event){
+  event.preventDefault();
+  header = $(this).closest('.form-div').find('#review_header')
+  body = $(this).closest('.form-div').find('#review_body')
+  rating = $(this).closest('.form-div').find('#review_rating')
```

Comment:
Mixed double and single quotes.<br>Missing semicolon.<br>'rating' is not defined.